src/background.js

```diff
@@ -65,20 +76,30 @@ function handleWake() {
   const now = Date.now();
   log('woke at', now);
   return browser.storage.local.get().then(items => {
-    const due = Object.values(items).filter(item => item.time <= now);
+    const due = Object.entries(items).filter(entry => entry[1].time <= now);
```

Comment:
Oh, wait!  I don't think we can, because we use `due.map(entry => entry[0])` down on line 100, so `due` needs to be the entries…